librz/bin/format/pe/pe_section.c

```diff
@@ -243,28 +243,19 @@ void PE_(write_image_section_header)(RzBuffer *b, ut64 addr, PE_(image_section_h
 struct rz_bin_pe_section_t *PE_(rz_bin_pe_get_sections)(RzBinPEObj *bin) {
 	struct rz_bin_pe_section_t *sections = NULL;
 	PE_(image_section_header) * shdr;
-	int i, j, section_count = 0;
+	int i, j;
 	char sec_name[PE_IMAGE_SIZEOF_SHORT_NAME + 1];
 
 	if (!bin || !bin->nt_headers) {
 		return NULL;
 	}
```

Comment:
I think it can't ever be, because `num_sections` is initialized with `	bin->num_sections = bin->nt_headers->file_header.NumberOfSections;` in `bin_pe_init_sections` and `NumberOfSections` is a ut16, while `num_sections` is a int. Thus there will never be the value `0xffffffff` in there.